clang/test/SemaCXX/cxx2b-warn-shadow.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -verify -fsyntax-only -std=c++2b -Wshadow-all %s
+
+namespace GH95707 {
+struct Foo {
+  int a; // expected-note {{previous declaration is here}}
+
+  void f1(this auto &self, int a) { self.a = a; }
+  void f2(int a) { } // expected-warning {{declaration shadows a field of 'GH95707::Foo'}}
+  void f3() {
+    [&](this auto &self, int x) { }; // expected-warning {{expression result unused}}
```

Comment:
```suggestion
    (void) [&](this auto &self, int a) { };
```
Typo, and add `(void)` so we don’t get a warning we don’t care about in this case ;þ